stdlib/asyncio/base_events.pyi

```diff
@@ -184,7 +200,7 @@ class BaseEventLoop(AbstractEventLoop, metaclass=ABCMeta):
         ) -> _TransProtPair: ...
     if sys.version_info >= (3, 7):
         async def sock_sendfile(
-            self, sock: socket, file: IO[bytes], offset: int = ..., count: Optional[int] = ..., *, fallback: bool = ...
+            self, sock: socket, file: IO[bytes], offset: int = ..., count: Optional[int] = ..., *, fallback: Optional[bool] = ...
```

Comment:
Why do you need this change? In the implementation I don't see any indication `None` is accepted; the default value is `True`.

Reply:
Oh it defaults to None in events.py. Seems like a mistake in the code, but the Optional here is harmless, so let's not worry about it.